src/lib/snark_worker/prod.ml

```diff
@@ -126,14 +126,16 @@ module Inputs = struct
                                     , `Sparse_ledger w.second_pass_ledger
                                     , `Connecting_ledger_hash
                                         input.connecting_ledger_left
-                                    , zkapp_command )
+                                    , Zkapp_command.write_all_proofs_to_disk
+                                        zkapp_command )
                                   ]
                                 |> List.rev )
                             |> Result.map_error ~f:(fun e ->
                                    Error.createf
                                      !"Failed to generate inputs for \
                                        zkapp_command : %s: %s"
-                                     ( Zkapp_command.to_yojson zkapp_command
```

Comment:
Change in logging format in snark worker, will appear only in case of unexpected error, transaction is given for debugging purposes.